lib/palindrome_check.rb

```diff
@@ -1,7 +1,26 @@
 # A method to check if the input string is a palindrome.
 # Return true if the string is a palindrome. Return false otherwise.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n), where n is the length of the array
```

Comment:
Why?

Also, under the covers, a string is just an array of characters. However, I recommend making sure to distinguish between "array" and "string" in interviews. I got harrassed by an interviewer one time because I kept saying "string" instead of "array", even though I was clearly using the right concept on the board. 😆